source_code/main_mcu/src/main.c

```diff
@@ -472,20 +491,20 @@ int main(void)
     /* Start animation */    
     if ((BOOL)custom_fs_settings_get_device_setting(SETTINGS_BOOT_ANIMATION) != FALSE)
     {
-        for (uint16_t i = GUI_ANIMATION_FFRAME_ID; i < GUI_ANIMATION_NBFRAMES; i++)
-        {
+    for (uint16_t i = GUI_ANIMATION_FFRAME_ID; i < GUI_ANIMATION_NBFRAMES; i++)
+    {
             timer_start_timer(TIMER_WAIT_FUNCTS, 28);
-            sh1122_display_bitmap_from_flash_at_recommended_position(&plat_oled_descriptor, i, FALSE);
-            while(timer_has_timer_expired(TIMER_WAIT_FUNCTS, TRUE) == TIMER_RUNNING)
-            {
-                comms_aux_mcu_routine(MSG_RESTRICT_ALL);
-                
+        sh1122_display_bitmap_from_flash_at_recommended_position(&plat_oled_descriptor, i, FALSE);
+        while(timer_has_timer_expired(TIMER_WAIT_FUNCTS, TRUE) == TIMER_RUNNING)
+        {
+            comms_aux_mcu_routine(MSG_RESTRICT_ALL);
+    
                 /* Click to exit animation */
                 if (inputs_get_wheel_action(FALSE, FALSE) == WHEEL_ACTION_SHORT_CLICK)
-                {
+    {
```

Comment:
indent error?

Reply:
Never touched this, but apparently I've messed this up during a merge.